azure_iot_edge/README.md

```diff
@@ -0,0 +1,114 @@
+# Agent Check: Azure IoT Edge
+
+## Overview
+
+[Azure IoT Edge][1] is a fully managed service to deploy Cloud workloads to run on Internet of Things (IoT) edge devices via standard containers.
+
+Use the Datadog-Azure IoT Edge integration to collect metrics and health status from IoT Edge devices.
+
+**Note**: this integration requires IoT Edge runtime version 1.0.10 or above.
+
+## Setup
+
+Follow the instructions below to install and configure this check for an IoT Edge device running on a device host.
+
+### Installation
+
+The Azure IoT Edge check is included in the [Datadog Agent][2] package.
+
+No additional installation is needed on your device.
+
+### Configuration
+
+It is recommended to configure the IoT Edge device so that the Agent runs as a custom module. Follow the official Microsoft documentation on [deploying Azure IoT Edge modules][3] for general information on installing and working with custom modules for Azure IoT Edge.
```

Comment:
```suggestion
Configure the IoT Edge device so that the Agent runs as a custom module. Follow the Microsoft documentation on [deploying Azure IoT Edge modules][3] for information on installing and working with custom modules for Azure IoT Edge.
```